tests/integration/api/test_observation_table.py

```diff
@@ -96,15 +112,15 @@ async def test_observation_table_from_view(event_table, scd_table, session, sour
         f"MY_OBSERVATION_TABLE_FROM_VIEW_{source_type}",
         sample_rows=sample_rows,
         columns=[view.timestamp_column, "ÜSER ID"],
-        columns_rename_mapping={view.timestamp_column: "POINT_IN_TIME"},
+        columns_rename_mapping={view.timestamp_column: "POINT_IN_TIME", "ÜSER ID": "üser id"},
     )
     assert observation_table.name == f"MY_OBSERVATION_TABLE_FROM_VIEW_{source_type}"
     check_location_valid(observation_table, session)
     await check_materialized_table_accessible(observation_table, session, source_type, sample_rows)
 
     check_materialized_table_preview_methods(
         observation_table,
-        expected_columns=["POINT_IN_TIME", "ÜSER ID"],
```

Comment:
@YS-L - with this validation, we will also be enforcing case sensitivity in column names. how do you feel about it?

Reply:
That seems fine to me since it should be a requirement already anyway.